src/libraries/System.Runtime/src/System.Runtime.csproj

```diff
@@ -3,6 +3,14 @@
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
   </PropertyGroup>
+  <PropertyGroup>
+    <!-- the System.Runtime project is special, we need to be able to hide the System.Private.Uri types for the generated ref .cs so we can include it in the System.Private.CoreLib ref .cs -->
+    <ContractTypesPartiallyMovedToUri>true</ContractTypesPartiallyMovedToUri>
```

Comment:
Can this be just `ContractTypesPartiallyMoved` instead of separate properties for CoreLib and Uri? From what I can see, this property is just a feature switch (target condition) and isn't used to infer the project.